src/main/java/seedu/address/logic/parser/DeletePersonCommandParser.java

```diff
@@ -3,26 +3,26 @@
 import static seedu.address.commons.core.Messages.MESSAGE_INVALID_COMMAND_FORMAT;
 
 import seedu.address.commons.core.index.Index;
-import seedu.address.logic.commands.DeleteCommand;
+import seedu.address.logic.commands.DeletePersonCommand;
 import seedu.address.logic.parser.exceptions.ParseException;
 
 /**
  * Parses input arguments and creates a new DeleteCommand object
```

Comment:
The same thing as above few comments, I think we should change documentation to reflect new classes.